front/src/utils/api/user.ts

```diff
@@ -0,0 +1,5 @@
+import { IUserData } from '../../interfaces/utils/api/IUserContext';
+
+export default function isTargetLoginedUser(loginUser: IUserData, targetUser: IUserData): boolean {
```

Comment:
로그인한 유저와 타겟 유저가 같은지를 검증하는 함수니까 
```suggestion
export default function isSameUser(loginUser: IUserData, targetUser: IUserData): boolean {
```
정도면 충분하지 않을까요??

아니면 
```suggestion
export default function isLoginedUser(loginUser: IUserData, targetUser: IUserData): boolean {
```
login이 중요한 포인트라면 target도 없어도 충분히 잘 읽을 수 있을 것 같습니다!

Reply:
아하 타겟 없이 isLoginedUser로 수정하겠습니다!